databases/week1/datab.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE
+	status CHANGE id id int unsigned NOT NULL;
+	select COUNT(*)from task;
+	--Find out how many tasks in the task table do not have a valid due date
+	SELECT * from task where due_date is NULL; 
+	--Find all the tasks that are marked as done.
+	SELECT * from task 
+    join status on task.status_id = status.id 
+    where status.name  = 'Done';
+	-- Find all the tasks that are not marked as done
+	SELECT * from task 
+    join status on task.status_id = status.id 
+    where status.id < 3;
+    --Get all the tasks, sorted with the most recently created first
+	SELECT MAX(created) from task GROUP BY user_id;
```

Comment:
Close to get it.
Try this way and you don't need to use the GROUP BY Clause. but ORDER BY
`SELECT title, created from task ORDER BY created DESC;`